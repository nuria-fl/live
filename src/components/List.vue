<template>
  <ul class="List">
    <li v-for="item in list" :key="item.id">
      <Item :item="item" />
    </li>
  </ul>
</template>

<script>
import Item from '@/components/Item'

export default {
  name: 'List',
  components: {
    Item
  },
  props: {
    list: {
      type: Array,
      required: true
    }
  }
}
</script>
