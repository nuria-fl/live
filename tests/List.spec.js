import { shallow } from 'vue-test-utils'

import List from '../src/components/List'

describe('List', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(List)
  })

  test('It should match snapshot', () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
