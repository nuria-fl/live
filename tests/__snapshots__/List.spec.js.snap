// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List It should match snapshot 1`] = `
<ul
  class="List"
/>
`;
